project/Newevaluation.html

```diff
@@ -110,253 +143,221 @@
-      font-size: 15px;
+      font-size: 1.1rem;
       color: grey;
-    }
+      }
+
+
+/* -------------------------------General Impression------------------------- */
 
     .thirdBox {
       background-color: whitesmoke;
-      margin-top: -21px;
-      width: 1895px;
-      height: 200px;
+      margin-top: 2rem;
+      width: 100%;
+      height: 12rem;
     }
 
+
+
+/* ------------------------Workflow, Leadership & Soft skills-----------------*/
+
     .fourthBox {
       background-color: whitesmoke;
-      margin-top: -21px;
-      width: 1895px;
-      height: 200px;
+      margin-top: 2rem;
+      width: 100%;
+      height: 12rem;
     }
 
+
+/* -------------------------------OOP, Design Patterns------------------------*/
+
     .fifthBox {
       background-color: whitesmoke;
-      margin-top: -21px;
-      width: 1895px;
-      height: 250px;
-      padding-top: 30px;
+      margin-top: 2rem;
+      width: 100%;
+      height: 16rem;
+      padding-top: 0rem;
     }
 
     legend {
-      //margin-left: 410px;
-      //background-color: whitesmoke;
-      font-size: 25px;
+      font-size: 1.5rem;
       color: #03A9F4;
-      //margin-bottom: -30px;
-      //font-weight: normal;
     }
 
     fieldset {
       border: 1px solid black;
-      width: 1130px;
+      width: 70.5rem;
       height: 67%;
-      margin-left: 400px;
-      //margin-top: 50px;
-      font-size: 20px;
+      margin-left: 25rem;
+      font-size: 1.3rem;
       color: rgb(82, 86, 89);
     }
 
     li {
       display: inline;
       float: left;
       list-style-type: none;
-      //margin-right: 118px;
     }
 
 
-  /*  label{
-      display: inline-block;
-      margin-left: 10px;
-      margin-right: 15px;
-      margin-top: 10px;
-      //padding-right: 50px;
+    li {
+      width: 29%;
+      margin-left: -4%;
+      margin-right: 8%;
+      margin-bottom: 2%;
     }
 
-    option, select{
-      width: 100px;
-      text-align: center;
-      margin-left: 70px;
-      margin-top: 20px;
-      display: inline-block;
-      //margin-right: 50px;
-      //padding-left: 50px;
-    }
+    select {
+      float: right;
 
-    select{
-    //  margin-left: 10%;
-    }*/
+    }
 
-    option, select{
-      font-size: x-large;
-      color: rgb(77, 77, 77);
-      width: 7%;
-      text-align: center;
-      margin-left: 4%;
-      margin-top: 1%;
-      margin-right: 5%;
-      display: inline-block;
+    option {
+      width: 2%;
     }
 
     label {
-      font-size: large;
-      display: inline-block;
-      color: rgb(77, 77, 77);
-      margin-left:-4%;
-      margin-right: 4%;
-      margin-top: 1%;
-      width: 15%;
-      text-align: left;
-      font-family: roboto;
+      width: 10%;
     }
 
 
-    //.firstLi {
-    //  padding-left: 10px;
-      //margin-left: 0px;
-    //}
 
-    //.firstSel {
-    //  padding-left: 10px;
-    //}
+/* ------------------------------------ HTTP ---------------------------------*/
 
     .sixthBox {
```

Comment:
yes. I'm thinking about the :nth-child selector, but I didn't know about it before, I'll change my code. Lucky me I didn't have one hundred boxes :))